lib/pipeline_controller.py

```diff
@@ -36,10 +36,11 @@
     'creation_date_et': 'string',
     'deletion_date_et': 'string',
     'circ_active_date_et': 'string',
-    'ptype_code': 'Int64',
-    'pcode3': 'Int64',
+    'ptype_code': 'Int16',
```

Comment:
Why did you end up changing to Int16? Just curious

Reply:
Ah because even though they're stored as Int64 in Sierra they are actually very small integers and are stored in Redshift as SMALLINT. This was one of the first pollers I made so it's still Int64 but the others treat these fields as Int16.